mono/mini/method-to-ir.c

```diff
@@ -8408,10 +8428,13 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 				/* Handle tailcalls similarly to calls */
 				DISABLE_AOT (cfg);
 
+				MonoMethodSignature *sig = mono_method_signature (cmethod);
+
+				emit_tailcall_parameters (cfg, sig);
```

Comment:
There is already an fsig computed above.

Reply:
Agreed, fixed.